cluster/validate-cluster.sh

```diff
@@ -91,7 +93,12 @@ while true; do
   # Suppress errors from kubectl output because during cluster bootstrapping
   # for clusters where the master node is registered, the apiserver will become
   # available and then get restarted as the kubelet configures the docker bridge.
-  node=$(kubectl_retry get nodes) || continue
+  node=$(kubectl_retry get nodes) || if [[ "${attempt}" -gt "${MAX_ATTEMPTS}" ]]; then
```

Comment:
Can you make this simpler - for example don't mix assignment and if with exit and continue. Its hard to tell what this thing is exactly doing without good bash knowledge.